fast_pytorch_kmeans/multi_kmeans.py

```diff
@@ -146,7 +150,7 @@ def fit_predict(self, X, centroids=None):
     device = X.device.type
     start_time = time()
     if self.centroids is None:
-      self.centroids = X[:, np.random.choice(batch_size, size=[self.n_clusters], replace=False)]
+      self.centroids = init_methods[self.init_method](X, self.n_clusters, self.minibatch)

```

Comment:
in MultiKMeans, `X` is a 3-d tensor, whose shape is `[n_kmeans, n_data, n_features]`. So in this case, either `init_method` has to be performed in `n_kmeans` iterations, or the initialization algorithm should support initializing multiple kmeans in parallel.

Reply:
This is fixed by a shape check with extra exception raised